sdk/python/setup.py

```diff
@@ -77,7 +77,7 @@
 ]
 
 REDIS_REQUIRED = [
-    "redis-py-cluster==2.1.2",
+    "redis==4.1.*",
```

Comment:
thoughts on this being too restrictive? ideally, we'd support both, but users might be using other versions of redis locally too.

Reply:
i guess we can do this now and see if anybody complains

Reply:
maybe at least do a redis >= 4.1.0?